### ISSUE TITLE
[Feature request] Reaction icons to announcement

### ISSUE THREAD

**Issue by ArtificialOwl** [nextcloud/announcementcenter#474]
When browsing the listing of last announcements, could be nice to be able to react using the usual reaction icons, like already available on Talk.

**Comment by almereyda** [comment]
Yes, quick reactions are a suitable way for a large, mostly anonymous crowd to quickly show presence, reciprocity and their gratitude. 🙏🏽
